lib/carrierwave-aws.rb

```diff
@@ -29,6 +29,9 @@ class Base
       add_config :aws_write_options
       add_config :aws_acl
       add_config :aws_signer
+      add_config :asset_host_public
+
+      attr_writer :asset_public_host
```

Comment:
This isn't necessary, `add_config` is a macro that defines acessors automatically.